content/en/docs/concepts/storage/volumes.md

```diff
@@ -1155,6 +1155,21 @@ spec:
 
 More examples can be found [here](https://github.com/kubernetes/examples/tree/master/staging/volumes/vsphere).
 
+#### CSI Migration
+
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
```

Comment:
@xing-yang 
we are going with `alpha` as per https://github.com/kubernetes/kubernetes/pull/90911#discussion_r435529744
Once test results are posted, we will generate another PR to bump to `beta`

Reply:
Sure

Reply:
Done